dev/TEAL.md

```diff
@@ -201,6 +201,41 @@ _available_.
 
  * Since v7, the account associated with any contract present in the
    `txn.ForeignApplications` field is _available_.
+   
+ * Since v9, there is group-level resource sharing. Any resource that
+   is available in _some_ top-level transaction in a transaction group
+   is available in _all_ v9 or later application calls in the group,
+   whether those application calls are top-level or inner.
+   
+ * When considering whether an asset holding or application local
+   state is available by group-level resource sharing, the holding or
+   local state must be available in a top-level transaction without
+   considering group sharing. For example, if account A is made
+   available in one transaction, and asset X is made available in
+   another, group resource sharing does _not_ make A's X holding
+   available.
+     
+ * Top-level transactions that are not application calls also make
+   resources available to group-level resource sharing. The following
+   resources are made available other transaction types.
+
+     1. `pay` - `txn.Sender`, `txn.Receiver`, and
+        `txn.CloseRemainderTo` (if set)
+
+     1. `keyreg` - `txn.Sender`
```

Comment:
minor: replace 1,1,1,... by 1,2,3,4,...

Reply:
I think that markdown causes the numbers to come out consecutively, but I might be mixing md up with some other format. I'll check.

Reply:
Yes, repeated 1 is ok:
https://www.markdownguide.org/basic-syntax/#lists-1